Tests/Phakchi/IntegrateTest.swift

```diff
@@ -338,9 +338,15 @@ class IntegrateTestCase: XCTestCase {
 
     override func tearDown() {
         super.tearDown()
-        let exp = expectationWithDescription("session is closed")
-        session.close {
-            exp.fulfill()
+
+        let expectation = expectationWithDescription("session is closed")
+
+        // workaround to pass test on CI
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(1 * Double(NSEC_PER_SEC)))
```

Comment:
Shouldn't you just increase the timeout duration given to `waitForExpectationsWithTimeout`?